src/views/v2/month/calendar-body/day.php

```diff
@@ -76,7 +87,7 @@
 	>
 		<h3 class="tribe-events-calendar-month__day-date tribe-common-h6 tribe-common-h--alt">
 			<span class="tribe-common-a11y-visual-hide">
-				<?php echo esc_html( sprintf( _n( '%s event', '%s events', $day['found_events'], 'the-events-calendar' ), number_format_i18n( $day['found_events'] ) ) ); ?>,
+				<?php echo esc_html( $num_events_label); ?>,
```

Comment:
* :no_entry_sign: Expected 1 spaces before closing bracket; 0 found
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: 20f9efaab55d9998a28b0223a498fdcd</sub>